### DISCUSSION TITLE
Size of Bootstrap icon package 1.06mb ?

### DISCUSSION THREAD

**Discussion by skywalk1411** [twbs/bootstrap#37948] [Q&A] [answered]
Hello, 

I am looking for suggestions on how to use Bootstrap icons, I find that icon package is very heavy to use, it is 1.08mb to use the full package. I like the icons because they follow the theme and colors, If I just seems to point at the svg, my colors don't apply to the svg icon, so how am I supposed to use the icon package if I do not want to preload that 1mb icon package?

I am using 10 icons so far on my website, plan to use maybe another 10 more, but I would like not to use the 1mb package if possible.

You can look at my website https://social.stonkch.art or the screenshot I posted on twitter https://twitter.com/stonkch_art/status/1617709051865075714?s=20

thank you.

**Comment by mdo** [comment, marked as answer]
If you want to use the SVGs, but not load the entire sprite for 2,000-ish icons, you could create your own sprite like we do. Here’s a demo from our [color modes example](https://github.com/twbs/examples/tree/main/color-modes):

https://github.com/twbs/examples/blob/d4232f937a7e3ef954cae49df604b18e04a66829/color-modes/index.html#L11-L29

**Reply by skywalk1411** [reply]
If I use the way you suggested, will the theme apply? or the svg will be black when theme is dark?   thank you

**Reply by mdo** [reply]
Any online SVGs or icons fonts can be setup to inherit their color. Only SVGs used via `<img>` element don't get that behavior.
